api/api/src/main/scala/uk/ac/wellcome/platform/api/Main.scala

```diff
@@ -0,0 +1,58 @@
+package uk.ac.wellcome.platform.api
+
+import scala.concurrent.ExecutionContext
+
+import akka.Done
+import akka.actor.ActorSystem
+import akka.stream.ActorMaterializer
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.server.Directives._
+
+import com.sksamuel.elastic4s.Index
+
+import com.typesafe.config.Config
+
+import uk.ac.wellcome.typesafe.{Runnable, WellcomeTypesafeApp}
+import uk.ac.wellcome.typesafe.config.builders.AkkaBuilder
+import uk.ac.wellcome.typesafe.config.builders.EnrichConfig._
+import uk.ac.wellcome.elasticsearch.typesafe.ElasticBuilder
+import uk.ac.wellcome.elasticsearch.DisplayElasticConfig
+import uk.ac.wellcome.platform.api.models.ApiConfig
+
+object Main extends WellcomeTypesafeApp {
+
+  runWithConfig { config: Config =>
+    implicit val actorSystem: ActorSystem =
+      AkkaBuilder.buildActorSystem()
+    implicit val executionContext: ExecutionContext =
+      AkkaBuilder.buildExecutionContext()
+    implicit val materializer: ActorMaterializer =
+      AkkaBuilder.buildActorMaterializer()
+
+    val elasticClient = ElasticBuilder.buildElasticClient(config)
+
+    val elasticConfig =
+      DisplayElasticConfig(indexV2 = Index(config.required("es.index.v2")))
+
+    val apiConfig =
+      ApiConfig(
+        host = config.getOrElse[String]("api.host")(default = "localhost"),
+        scheme = config.getOrElse[String]("api.scheme")(default = "https"),
+        defaultPageSize = config.getOrElse[Int]("api.pageSize")(default = 10),
+        pathPrefix =
+          s"${config.getOrElse[String]("api.apiName")(default = "catalogue")}",
+        contextSuffix = config.getOrElse[String]("api.context.suffix")(
+          default = "context.json"),
+      )
+
+    val router = new Router(elasticClient, elasticConfig, apiConfig)
+
+    new Runnable {
+      def run() =
+        Http()
+          .bindAndHandle(router.routes, apiConfig.host, 8080)
```

Comment:
Reckon this means it just presents the same as the current app to the nginx proxy in the task definition. We should look later at making sure we are using the same nginx proxy image across the estate.

Reply:
Yea, this needs changing.

We use the container / sidecar setup, and nginx is running on port 9000:
https://github.com/wellcometrust/catalogue/blob/master/api/terraform/catalogue_api/variables.tf#L24-L26

Looking at the dockerfile we used to use and the terraform, we are on `8888`.
https://github.com/wellcometrust/catalogue/blob/master/api/terraform/main.tf#L9
https://github.com/wellcometrust/dockerfiles/blob/master/finatra_service_base/Dockerfile#L11

It would be good to use whatever is standard for AKKA HTTP, which looks like `8080`, but we also tend to use the pattern:
nginx => 9000
app => 9001

Not to fussed about what we do, but we do need to change this inline with Docker / Terraform / Code.

Reply:
I have updated this  to `8888` for now